providers/apple/session.go

```diff
@@ -20,7 +20,9 @@ const (
 )
 
 type ID struct {
-	Sub string `json:"sub"`
+	Sub            string `json:"sub"`
+	Email          string `json:"email"`
+	IsPrivateEmail bool   `json:"is_private_email"`
```

Comment:
I got around to testing this and I had to change this to ``json:"is_private_email,string"``